### PR TITLE
Fix tanglegram

### PR DESCRIPTION
**Pull request by rlhadas**
Updated rendering of tanglegrams and reconciliations to reduce overlap in text.

### CODE REVIEW

**Comment by ssantichaivekin** [comment]
Will look at it next week along with the gui issues!

File: empress/recon_vis/recon_viewer.py [hunk 1]
```diff
@@ -129,7 +130,7 @@ def _render_host_helper(fig: plot_tools.FigureWrapper, node: tree.Node, show_int
 
     if node.is_leaf():
         text_offset = (node_pos.x + render_settings.TIP_TEXT_OFFSET_X, node_pos.y)
-        fig.dot(node_pos, col=render_settings.HOST_NODE_COLOR)
+        # fig.dot(node_pos, col=render_settings.HOST_NODE_COLOR)
```
Comment (aremath) [inline empress/recon_vis/recon_viewer.py:133]: Probably should just remove this?

File: empress/recon_vis/recon_viewer.py [hunk 2]
```diff
@@ -150,7 +151,7 @@ def _render_host_helper(fig: plot_tools.FigureWrapper, node: tree.Node, show_int
         _render_host_helper(fig, node.right_node, show_internal_labels, font_size, host_tree)
 
 
-def _render_parasite(fig: plot_tools.FigureWrapper, parasite_tree: tree.Tree, recon_obj: recon.Reconciliation, host_lookup: dict, parasite_lookup: dict, show_internal_labels: bool, show_freq: bool, font_size: float):
+def _render_parasite(fig: plot_tools.FigureWrapper, parasite_tree: tree.Tree, recon_obj: recon.Reconciliation, host_lookup: dict, parasite_lookup: dict, show_internal_labels: bool, show_freq: bool, font_size: float, longest_host_name: int):
```
Comment (aremath) [inline empress/recon_vis/recon_viewer.py:154]: Split this up over multiple lines....

Comment (aremath) [inline empress/recon_vis/recon_viewer.py:154]: I'm not sure why this function exists. Couldn't we just call the helper directly using parasite_tree.root?

File: empress/recon_vis/recon_viewer.py [hunk 3]
```diff
@@ -369,7 +375,7 @@ def _calculate_font_size(num_tip_nodes: int):
     :return the font size for the tips and internal nodes of a tree
     """
     x = (render_settings.START_SIZE - num_tip_nodes) / render_settings.STEP_SIZE
-    return 1.5 * _sigmoid(x)
+    return 3.0 * _sigmoid(x)  # 3.0 is a magic value that can be adjusted
```
Comment (aremath) [inline empress/recon_vis/recon_viewer.py:378]: These magic values should probably go at the top of the file to make it clearer that they exist and how to adjust them.

File: empress/recon_vis/recon_viewer.py [hunk 4]
```diff
@@ -378,7 +384,7 @@ def _sigmoid(x: float):
     :param x: A number to be plugged into the function
     :return a sigmoid value based on the input value, x
     """
-    return (1 / (1 + math.e**(-x)))
+    return (1 / (1 + math.e**(-0.8*x)))  # 0.8 is a magic value that can be adjusted
```
Comment (aremath) [inline empress/recon_vis/recon_viewer.py:387]: Ditto

File: empress/recon_vis/tanglegram.py [hunk 5]
```diff
@@ -82,13 +84,15 @@ def _render_helper_host(fig, node, show_internal_labels):
         # create layout for current node
         node.layout = tree.NodeLayout()
         node.layout.col = min(right_layout.col, left_layout.col) - HORIZONTAL_SPACING
+        if node.right_node.is_leaf() or node.left_node.is_leaf():
```
Comment (aremath) [inline empress/recon_vis/tanglegram.py:87]: This should get a comment explaining what it does.

File: empress/recon_vis/tanglegram.py [hunk 6]
```diff
@@ -131,13 +135,15 @@ def _render_helper_parasite(fig, node, show_internal_labels):
         # create layout for current node
         node.layout = tree.NodeLayout()
         node.layout.col = max(left_layout.col, right_layout.col) + HORIZONTAL_SPACING
+        if node.right_node.is_leaf() or node.left_node.is_leaf():
```
Comment (aremath) [inline empress/recon_vis/tanglegram.py:138]: This `if` should get a comment too.

**Pull request merged**